js/index.js

```diff
@@ -34,3 +35,9 @@ $navLinks.forEach(($navLink) => $navLink.addEventListener('click', handleNavLink
 
 const $logo = document.querySelector('#logo');
 $logo?.addEventListener('click', handleLogoClick);
+
+const $orderOpenButton = document.querySelector('#order-open');
+$orderOpenButton?.addEventListener('click', () => toggleModal());
```

Comment:
```suggestion
const $orderOpenButton = document.querySelector('#order-open');
$orderOpenButton?.addEventListener('click', handleOrderOpenClick);
```

Например бургер он действительно переключает (toggle) открытие/закрытие
А тут подразумевается только открытие